templates/summit/index.html

```diff
@@ -42,8 +44,8 @@ <h3 class="p-heading--2">Online</h3>
           <iframe width="560"
                   height="315"
                   class="u-embedded-media__element"
-                  src="https://www.youtube.com/embed/aR-yTFtjGtw?si=VdJuwA6Uq4u2kEiV&autoplay=1&mute=1&loop=1&playlist=aR-yTFtjGtw"
-                  title="Ubuntu Summit 2024 Highlights"
+                  src="https://www.youtube.com/watch?v=gKMz7FVPJjo"
```

Comment:
This should be an embed link, not a direct YouTube link - it currently fails to load on the page.